src/com/baloise/sharedlib/tools/internal/MavenImpl.groovy

```diff
@@ -0,0 +1,27 @@
+package com.baloise.sharedlib.tools.internal
+
+import java.nio.file.*
+
+import com.baloise.sharedlib.common.Registered
+import com.baloise.sharedlib.tools.Maven
+
+class MavenImpl extends Registered implements Maven{
+
+	@Override
+	// tag see https://hub.docker.com/_/maven
+	public void mvn(String cmd, String tag = "") {
+		String image = tag ? "maven:$tag" : "maven"
+		if(steps.isUnix()) {
+			steps.docker.image(image).inside('-v $HOME/.m2:/root/.m2') {
```

Comment:
you know that our current jenkins instances are not capable of running the docker plugin?

Reply:
podman does currently not support all of the commands and params that the docker plugin needs

Reply:
is the docker - incapability something we want to keep for the future? shall we use podman instead of docker?
Have you tried https://github.com/mgoltzsche/jenkins-jnlp-slave?
I would leave the docker decision and implementation as work in progress for now. The more interesting topic is Deployment IaC via Mycloud and the statement "let's try to use containers instead of jenkins global tools" is OK for me.